ratelimits/names.go

```diff
@@ -76,6 +76,9 @@ const (
 	// Note: When this is referenced in an overrides file, the fqdnSet MUST be
 	// passed as a comma-separated list of domain names.
 	CertificatesPerFQDNSet
+
+    // TODO: <Add Description> @kruti-s
+	IssuancePausedPerDomainPerAccount
```

Comment:
nit: I think this name is somewhat misleading -- this limit isn't counting the number of times that issuance has been paused. I think it should probably be named "FailedAuthorizations(SomeQualifier)PerDomainPerAccount", since it is a FailedAuthorizations limit at the end of the day.